core/lib/generators/spree/install/install_generator.rb

```diff
@@ -111,6 +111,7 @@ def include_seed_data
 \n
 Spree::Core::Engine.load_seed if defined?(Spree::Core)
 Spree::Auth::Engine.load_seed if defined?(Spree::Auth)
+SpreeStoreCredits::Engine.load_seed if defined?(SpreeStoreCredits::Engine)
```

Comment:
seeds should be part of core not changing this generator, and same for the assets below